packages/pwa-kit-runtime/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## v3.9.0-dev (Oct 29, 2024)
+- Consider CSP header changes in service worker etag generation [#2191](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2191)
```

Comment:
I think better if the changelog explains the bug that impacts the end user, rather than explaining etag and other implementation detail. How about this?

```suggestion
- Fix stale service worker file that could cause requests to still use old Content-Security-Policy [#2191](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2191)
```